lib/exporter/export_quran_meta_data.rb

```diff
@@ -55,7 +55,7 @@ def export_surah(format:)
         Chapter.order('id asc').each do |chapter|
           json_data[chapter.id] = {
             id: chapter.id,
-            name: chapter.name,
+            name: chapter.name_complex,
```

Comment:
Is there a benefit to this field? I see that it matches `chapter.id`

Reply:
That was a bug, try the updated JSON file now it hold the Surah name with diacritic.

Reply:
Just downloaded the json. Seeing the following. Does the json / sqlite need to be regenerated? I go an email that `ayah` resource updated but not for `Surah names`.

```
{
  "1": {
    "id": 1,
    "name": 1,
    "name_simple": "Al-Fatihah",
    "name_arabic": "الفاتحة",
    "revelation_order": 5,
    "revelation_place": "makkah",
    "verses_count": 7,
    "bismillah_pre": false
  },
  "2": {
    "id": 2,
    "name": 2,
    "name_simple": "Al-Baqarah",
    "name_arabic": "البقرة",
    "revelation_order": 87,
    "revelation_place": "madinah",
    "verses_count": 286,
    "bismillah_pre": true
  },

```

Reply:
JAK for the reminder, reexported the surah metadata and now name field will have Surah names with diacritic. 
 
```
{
"id":1,
"name":"Al-Fātiĥah",
"name_simple":"Al-Fatihah",
"name_arabic":"الفاتحة",
"revelation_order":5,
"revelation_place":"makkah",
"verses_count":7,
"bismillah_pre":false
}
```

Reply:
Awesome. Wanted to say Jazak Allah for this amazing resource!